esp/services/ws_workunits/ws_workunitsAuditLogs.cpp

```diff
@@ -1406,6 +1406,42 @@ void CWsWorkunitsSoapBindingEx::createAndDownloadWUZAPFile(IEspContext& context,
     request->getParameter("WhereSlow", zapInfoReq.whereSlow);
     request->getParameter("IncludeThorSlaveLog", zapInfoReq.includeThorSlaveLog);
     request->getParameter("ZAPFileName", zapInfoReq.zapFileName);
+
+    StringBuffer sendEmailStr, attachZAPReportToEmailStr, portStr;
+    request->getParameter("SendEmail", sendEmailStr);
+    zapInfoReq.sendEmail = atoi(sendEmailStr.str());
```

Comment:
is this typesafe (might some compilers complain?) . Maybe more like this?
zapInfoReq.sendEmail = atoi(sendEmailStr.str()) ?  true : false;

Reply:
I will change this.